### PR TITLE
`Init` functionality in `/packages/dapp/src/pages/index.tsx` 

### PR DESCRIPTION
**Pull request by Nazeeh21**
I've added all the required code for initializing the `init()` function.
Currently, I'm getting `provider=undefined` and thus it isn't allowing the init() functionality to run. I am trying to debug that and find a fix for that.

### CODE REVIEW

File: packages/dapp/helpers/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import DRecruiter from '../contracts/hardhat_contracts.json'
+import { ethers } from 'ethers'
+import { abis } from "./abi";
+
+export const loadDRecruitV1Contract = async (targetNetwork, signer) => {
+  console.log('Loading DRecruit contract...')
+  console.log(`Target network: ${targetNetwork}`)
+  const contract = new ethers.Contract(
```
Comment (damianmarti) [inline packages/dapp/helpers/index.js:8]: Why we need this load contract function if we are loading the contract here https://github.com/moonshotcollective/recruiter.party/pull/6/files#diff-b03267768cd4f21d635cd49366ee12debcbe6ccb425a314bbd8806fe08fb612bL176 ?

**Comment by damianmarti** [comment]
@Nazeeh21 When I load the home page with mumbai network selected in my connected account, I'm still getting a network error:

localhost_3000_ (1)

File: packages/dapp/helpers/index.js (hunk 1 above)
Comment (Nazeeh21) [reply packages/dapp/helpers/index.js:8]: We no longer need to load the contract now as Huxwell is storing in `Web3Context`, now we just need to use it from there

**Comment by Nazeeh21** [comment]
Currently, I'm integrating the `init` functionality, I'll have a look to it once I'll complete that

File: packages/dapp/helpers/index.js (hunk 1 above)
Comment (damianmarti) [reply packages/dapp/helpers/index.js:8]: So we can remove this load functions.

**Comment by Nazeeh21** [comment]
This PR is deprecated because this branch contains a lot of unnecessary code.
New PR can be found at https://github.com/moonshotcollective/recruiter.party/pull/14

**Pull request closed without merging**